### DISCUSSION TITLE
References from discussions in issues/PRs and visa-versa

### DISCUSSION THREAD

**Discussion by lindsayad** [community/community#56] [Discussions] [answered]
It appears that right now if you reference an issue/PR from a discussion, there is no indication of that reference on the referenced issue/PR. Similarly if you reference a discussion from an issue/PR, then the referenced discussion shows no indication that it was referenced. It would be great if references involving discussions work the same way as existing references between issues/pulls. This really helps in preventing bifurcation and repetition of conversation. Someone coming to a discussion which was referenced by an issue/pull would be encouraged to click over to that issue/pull (and visa versa) before saying something that has perhaps already been said.

**Comment by becca** [comment, marked as answer]
This is on our roadmap to improve the experience so you can more easily follow where Discussions are referenced. Will keep you updated when we can land a solution for this. Thank you for the feedback 🚀

**Reply by marcindulak** [reply]
There is a newer instance of this feature request https://github.com/orgs/community/discussions/2971.
